tests/test_tracking_grid/test_tracking_grid.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TestHarness
+from input_set import GridInput
+
+import openmoc
```

Comment:
Per PEP8, put the `math` import at the top along with the other standard Python modules `os` and `sys`, and before third party modules like `openmoc`